sources/engine/Xenko.VirtualReality/TrackedDevice.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Xenko contributors (https://xenko.com) and Silicon Studio Corp. (https://www.siliconstudio.co.jp)
+// Distributed under the MIT license. See the LICENSE.md file in the project root for more information.
+using System;
+using Xenko.Core.Mathematics;
+using Xenko.Games;
+
+namespace Xenko.VirtualReality
+{
+    public abstract class TrackedDevice : IDisposable
+    {
+        public abstract Vector3 Position { get; }
+
+        public abstract Quaternion Rotation { get; }
+
+        public abstract Vector3 LinearVelocity { get; }
+
+        public abstract Vector3 AngularVelocity { get; }
+
+        public abstract DeviceState State { get; }
+
+        public abstract DeviceClass Class { get; }
+
+        public abstract string SerialNumber { get; }
+
+        public virtual void Update(GameTime time)
+        {           
+        }
+
+        public virtual void Dispose()
```

Comment:
If it doesn't do anything then either implementing `IDisposable` is not required or it should follow the proper pattern:
```csharp
public void Dispose()
{
   Dispose(true);
}

protected abstract void Dispose(bool disposing)
{
}
```

Reply:
It's not being done on any other input classes, so for consistency let's keep it that way for now.